apps/webbsite/src/pages/api/subscribers/index.ts

```diff
@@ -47,6 +47,7 @@ export default async function handler(
     },
     body: JSON.stringify({
       attributes: { FIRSTNAME: name },
+      listIds: [5],
```

Comment:
Please add a todo here `TODO: Extract the listId to be the environment variable`.